distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/restito/ChunkedContent.java

```diff
@@ -151,5 +237,52 @@ private Response respond(Response response) {
 
             return response;
         }
+
+        /**
+         * Holds byte offset data specified by the request's range header and any related parsing
+         * methods.
+         */
+        private class ByteOffset {
+            public int start;
+            public int end;
+
+            public ByteOffset(int start, int end){
+                this.start = start;
+                this.end = end;
+            }
+
+            /**
+             * Tokenizes the starting and ending bytes values from a range header. The stub server only
+             * advertising supporting byte offsets specifically, so this function assumes that byte
+             * ranges are provided and does not evaluate the unit of measurement
+             * @param rangeHeaderValue          Value of the range header sent in the request
+             * @param totalSizeOfProductInBytes Total size of the message to be returned. This is
+             *                                  not the partial content size, but the FULL size of
+             *                                  the product.
+             */
+            public ByteOffset(String rangeHeaderValue, int totalSizeOfProductInBytes){
+                // extract bytes
```

Comment:
Awesome, I was also able to use it's substringAfter() method for the next statement as well. Thanks!